src/main/java/wooteco/subway/member/dao/MemberDao.java

```diff
@@ -39,7 +41,8 @@ public Member insert(Member member) {
 
     public void update(Member member) {
         String sql = "update MEMBER set email = ?, password = ?, age = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
+        jdbcTemplate.update(sql,
+            new Object[]{member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
```

Comment:
```suggestion
        jdbcTemplate.update(sql, member.getEmail(), member.getPassword(), member.getAge(), member.getId());
```

(제공된 부분이긴 하지만)
`update(String sql, @Nullable Object... args)` 메서드도 있기 때문에 그냥 넘겨줘도 되겠네요~